etc/picongpu/frontier-ornl/batch_picongpu.profile.example

```diff
@@ -65,6 +64,11 @@ module load freetype/2.11.1
 #
 # module load openpmd-api/0.15.X
 
+# Please install cmake on your own and put it in your PATH.
+# ORNL is only providing outdated modules.
+#
+# export PATH=$HOME/cmake-3.30.3-linux-x86_64/bin:$PATH
```

Comment:
Using 
```
module load Core/24.07
module load cmake/3.27.9
```
after loading all other modules provides a working and p-to-date cmake. (I tested comiling but **without** png, openpmd, fftw3, and isaac)

Reply:
> Using
> 
> ```
> module load Core/24.07
> module load cmake/3.27.9
> ```
> 
> after loading all other modules provides a working and p-to-date cmake. (I tested comiling but **without** png, openpmd, fftw3, and isaac)

I did that too, but this broke the git and freetype modules. I did the compiling with openpmd, fftw3, but yet without isaac.

Reply:
Note: Yesterday, I found out that the profile shown here does not compile anymore unless you bump up the rocm-version to rocm/6.2.0. I will include this in the next commit.

Reply:
We need at least CMake 3.25 therefore he added it.

Reply:
btw. we increased the CMake version because of an feature which will now not be used. I am on the way to relax the cmake version back to 3.22

Reply:
#5113 is relaxing the cmake version back to 3.22